src/shared/modules/user/user.model.ts

```diff
@@ -0,0 +1,35 @@
+// unused. HERE FOR EXAMPLE!!!!
+
+import { Schema, Document, model } from 'mongoose';
+import { UserInterface } from '../../types/index.js';
+
+export interface UserDocument extends UserInterface, Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const userSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      minlength: [2, 'Min length for name is 2'],
```

Comment:
export const MIN_NAME_LENGTH = 2;
export const MIN_NAME_LENGTH_ERROR_MESSAGE = `Min length for name is ${MIN_NAME_LENGTH}` 

minlength: [MIN_NAME_LENGTH , MIN_NAME_LENGTH_ERROR_MESSAGE],